app/controllers/avo/base_controller.rb

```diff
@@ -358,10 +358,10 @@ def set_actions
       @actions = @resource
         .get_actions
         .map do |action|
-          action[:class].new(record: @record, resource: @resource, view: @view, arguments: action[:arguments])
+          action.is_a?(DividerComponent) ? [action] : action[:class].new(record: @record, resource: @resource, view: @view, arguments: action[:arguments], icon: action[:icon])
```

Comment:
```suggestion
          action[:class].new(record: @record, resource: @resource, view: @view, arguments: action[:arguments], icon: action[:icon])
```

Can we do something like this? The `DividerComponent` initialiser will ignore this arguments.

Reply:
I will update it.